lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -214,6 +215,17 @@ SectionHeader WasmBinaryReader::ReadSectionHeader()
     return header;
 }
 
+#if ENABLE_DEBUG_CONFIG_OPTIONS
+Js::FunctionBody* WasmBinaryReader::GetFunctionBody() const
+{
+    if (m_readerState == READER_STATE_FUNCTION)
+    {
+        return m_funcState.body;
+    }
+    return nullptr;
+}
+#endif
+
 #if DBG_DUMP
 void WasmBinaryReader::PrintOps()
```

Comment:
I wasn't sure if we wanted to keep PrintOps.
In the current code base in doesn't do anything, because we print before actually parsing any opcode.